src/app/organization/offices/create-offices/create-offices.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { OrganizationService } from '../../organization.service';
+import * as _moment from 'moment';
```

Comment:
Don't use `moment`, follow what is being done as of now within the other components as per this guide https://github.com/openMF/web-app/wiki/Date-and-Language